aten/src/ATen/cpu/vec256/vec256_qint.h

```diff
@@ -405,8 +405,8 @@ struct Vec256<c10::qint32> {
         Vec256<c10::qint32> zero_point,
         Vec256<c10::qint32> q_six) {
 #ifdef __AVX2__
-      return _mm256_min_epi8(
-          _mm256_max_epi8(vals, zero_point.vals), q_six.vals);
+      return _mm256_min_epi32(
+          _mm256_max_epi32(vals, zero_point.vals), q_six.vals);
```

Comment:
For int32, the only value of zero-point supported is 0, can we add an explicit check for that?

Reply:
Not here, no, this is used in the tight loop of the inner kernel

Why do you want this check? Correctness? It wouldn't help performance at all

Reply:
Not in the kernel, do we have a check for relu/relu6 that checks for this? The place to do this would be right the beginning of the function call: https://github.com/pytorch/pytorch/pull/25545/files#diff-a04b434d724c12b848bd2bc5d8dadd40R15

Reply:
That check doesn't belong there either, as mentioned in the README that file is only for things that differ depending on the CPU architecture. We can add this check to e.g. `qrelu.cpp` but I don't think this is relevant for this PR